### PR TITLE
logging and proctitle in tests

### PR DESCRIPTION
**Pull request by arvidn**
when running multiple services in the same process (in tests), don't initialize logging for all of them, and don't set the proctitle of the test

### CODE REVIEW

**Comment by altendky** [comment]
I like what's here but I think we need to set the tests to configure logging as well?  I thought I did a hacky thing in this direction at some point and ended up getting no logging until i made the tests initialize it.  Assuming I am remembering correctly, I'm not sure offhand what's best to handle this.  Could be a session scope auto-use fixture.  Could add a pytest plugin file and use one of those hooks.

**Comment by arvidn** [comment]
I see. I would have thought logging would happen by default

**Comment by altendky** [comment]
I might be wrong here.  Though even if there is a default handler we may want to get our regular formatting for familiarity.

**Comment by arvidn** [comment]
I confirmed that tests still produce logs, and that there are no duplicates. So I think this is a clear improvement over status quo.

File: chia/server/start_service.py [hunk 1]
```diff
@@ -51,7 +52,7 @@ def __init__(
         rpc_info: Optional[Tuple[type, int]] = None,
         parse_cli_args=True,
         connect_to_daemon=True,
-        handle_signals=True,
+        running_new_process=True,
```
Comment (altendky) [inline chia/server/start_service.py:55]: I might go with `manage_process_state` or something perhaps?  I also somewhat wonder if this should just always be handled outside of the service.  But, I'm excited to have this improvement.

**Review by altendky — DISMISSED** [review]

File: chia/server/start_service.py (hunk 1 above)
Comment (arvidn) [reply chia/server/start_service.py:55]: yeah, I agree that it probably belongs closer to the code spawning new processes, rather than here

**Review by altendky — APPROVED** [review]

**Pull request merged**